pkgs/games/brainworkshop/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, lib, fetchFromGitHub, python36 }:
```

Comment:
Is there a reason to hardcode `python36` instead of just `python3` ?

Reply:
This is a workaround as python3 is 3.8 something, which breaks pyglet (https://github.com/pvcraven/arcade/issues/514)